lib/events.js

```diff
@@ -56,6 +56,10 @@ Object.defineProperty(EventEmitter, 'defaultMaxListeners', {
     // force global console to be compiled.
     // see https://github.com/nodejs/node/issues/4467
     console;
+    // check whether the input is a positive number (whose value is zero or
+    // greater and not a NaN).
+    if (typeof arg !== 'number' || arg < 0 || arg !== arg)
+      throw new TypeError('defaultMaxListeners must be a positive number');
```

Comment:
'defaultMaxListeners' should be enclosed in double quotes to match the style when referencing actual property names.